src/DotNet/Importer.cs

```diff
@@ -52,6 +59,7 @@ public struct Importer {
 		readonly GenericParamContext gpContext;
 		RecursionCounter recursionCounter;
 		ImporterOptions options;
+		public IImportMapper Mapper;
```

Comment:
It should be readonly and not public. Add a new ctor with an extra IImportMapper arg.